p/pint/pint_ubi_8.5.sh

```diff
@@ -0,0 +1,71 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: pint
+# Version	: 0.18
+# Source repo	: https://github.com/hgrecco/pint
+# Tested on	: UBI 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=pint
+PACKAGE_VERSION=${1:-0.18}
+PACKAGE_URL=https://github.com/hgrecco/pint
+
+yum install -y git python38 wget
+
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-ppc64le.sh
+bash Miniconda3-latest-Linux-ppc64le.sh -b -p /conda
+export PATH=$PATH:/conda/bin
+conda update -y -n base conda
+conda create -n pint -y python=3.8
+eval "$(/conda/bin/conda shell.bash hook)"
+conda init bash
+conda activate pint
+
```

Comment:
Any Specific reason for using Anaconda package manager?

Reply:
@Ryan-Dmello Actually I tried it with **pip** earlier, but pip had to build many dependencies from source `setup.py install`, since wheels were not available. Building those was either taking a long time, or resulted in failure (due to multiple sub-dependencies, etc.)

Main issue was for building **llvmlite** package from source, which needed **LLVM** compiler/toolkit version 11.x.x. This was not available in the repos, and needed to be built from source as well. The building process takes around 3-4 hours, and also (sometimes) exhausts memory at the very end (around 97-98%).

Hence, I decided to use **conda** as it had had all the wheels ready (via **conda-forge**).